ui/src/viewers/HtmlViewer.jsx

```diff
@@ -1,27 +1,82 @@
 import React, { Component } from 'react';
+import DOMPurify from 'dompurify'
 
 import { Skeleton } from 'components/common';
 
 import './HtmlViewer.scss';
 
-class HtmlViewer extends Component {
+const FORBID_TAGS = [
+  "meta",
```

Comment:
`meta`, `base`, `bgsound`, `embed`, `frame`, `frameset`, `iframe`, `link`, `meta`, `object`, `plaintext`, `svg` are not in DOMPurify’s default allow list, so it’s probably redundant to specify them here, right?

https://github.com/cure53/DOMPurify/blob/main/src/tags.js